src/test/script_tests.cpp

```diff
@@ -101,6 +101,40 @@ static ScriptErrorDesc script_errors[]={
     {SCRIPT_ERR_WITNESS_PUBKEYTYPE, "WITNESS_PUBKEYTYPE"},
     {SCRIPT_ERR_OP_CODESEPARATOR, "OP_CODESEPARATOR"},
     {SCRIPT_ERR_SIG_FINDANDDELETE, "SIG_FINDANDDELETE"},
+    // Elements
+    {SCRIPT_ERR_ERROR_COUNT, "ERROR_COUNT"},
```

Comment:
As I mentioned in #1285, this isn't a real error, and it should probably be removed from this list.

Reply:
I can remove `SCRIPT_ERR_ERROR_COUNT` in my PR, but ultimately this has to be fixed on master. We could upstream my commit.